config.inc.php

```diff
@@ -137,8 +139,13 @@
 $i--;
 
 /* Uploads setup */
-$cfg['UploadDir'] = '';
-$cfg['SaveDir'] = '';
+if (isset($_ENV['PMA_UPLOADDIR'])) {
+    $cfg['UploadDir'] = trim($_ENV['PMA_UPLOADDIR']);
```

Comment:
I am not sure trim is needed, the user is responsible for the value. Right?

Reply:
Agree, user should put the string in correctly in the first place but I saw trim used around PMA_ABSOLUTE_URI so I replicated the pattern. Happy for you to make the call, @williamdes?

Reply:
Let's not replicate the trim for a folder path. Not sure why it was done for an url

Reply:
@williamdes, I've removed trim().